project/circles/templates/circles/circle_detail.html

```diff
@@ -103,6 +103,42 @@ <h2>
 
     {% include "circles/circle_invite_companion_modal.html" %}
 
+    <!--Pagination-->
+    <nav aria-label="Page navigation example">
+        <ul class="pagination justify-content-center">
+          {% if activity_page.has_previous %}
+            <li class="page-item">
+              <a class="page-link" href="?page={{ activity_page.previous_page_number }}">Previous</a>
+            </li>
+          {% else %}
+            <li class="page-item disabled">
+              <a class="page-link" href="#" tabindex="-1" aria-disabled="True">Previous</a>
```

Comment:
Remember the i18n tag here.
```suggestion
              <a class="page-link" href="#" tabindex="-1" aria-disabled="True">
                  {% translate "Previous" %}
              </a>
```